wp-content/themes/core/pcss/components/_dialog.pcss

```diff
@@ -0,0 +1,87 @@
+/* -----------------------------------------------------------------------------
+ *
+ * Component: Dialog
+ *
+ * ----------------------------------------------------------------------------- */
+
+.c-dialog[aria-hidden='true'] {
+	display: none;
+}
+
+.c-dialog__overlay {
+	background-color: var(--color-goethite);
+	bottom: 0;
+	color: var(--color-white);
+	left: 0;
+	overflow: scroll;
+	position: fixed;
```

Comment:
For the sake of robusticity (new made up word) I prefer to have a wrapper inside of a parent fixed container that handles overflow scrolling sort of stuff. Gives us a greater degree of dependability on this fixed container may or may not interact in positioning when other site areas come into play.